pyproject.toml

```diff
@@ -27,14 +27,16 @@ pre-commit = "^2.17.0"
 responses = "^0.24.1"
 flake8 = "^6.1.0"
 isort = "^5.12.0"
-mypy = "^1.7.1"
-types-requests = "^2.31.0.10"
+mypy = "^1.10.1"
+types-requests = "^2.32"
 pytest-cov = "^4.1.0"
 
 [tool.mypy]
 plugins = ["pydantic.mypy"]
 exclude = ["example/*"]
 
+[tool.black]
+target-version = ["py38"]
```

Comment:
Hmm... why `py38`?

Reply:
Unless I'm not up to date, it's the least version the SDK supports.

Reply:
But then shouldn't we also include 3.9, 3.10, etc. here? Why _only_ 3.8?

Reply:
Because we only ship one codebase to all of the versions and we should use the lowest common denominator.